lib/tests/streamlit/elements/data_editor_test.py

```diff
@@ -392,3 +390,26 @@ def test_with_unsupported_index(self, index: pd.Index):
 
         with self.assertRaises(StreamlitAPIException):
             st.experimental_data_editor(df)
+
+    @parameterized.expand(
+        [
+            (pd.RangeIndex(0, 3, 1),),
```

Comment:
Nit: ` (pd.RangeIndex(0, 3, 1),)`
Is there a reason why there is an extra comma?

Reply:
Parameterized requires this to be a tuple. And without the comma, it would not be a valid tuple.